stripe-core/src/main/java/com/stripe/android/core/exception/ExceptionUtils.kt

```diff
@@ -0,0 +1,12 @@
+@file:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+
+package com.stripe.android.core.exception
+
+import androidx.annotation.RestrictTo
+
+@get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+val Throwable.safeAnalyticsMessage: String
+    get() = when (this) {
+        is StripeException -> analyticsValue() ?: javaClass.name
+        else -> javaClass.name
+    }
```

Comment:
I made something similar to what iOS does in their [analytics client](https://github.com/stripe/stripe-ios/blob/0ae7a87cee200521f426e23cc5dc008a4dd4930b/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/STPAnalyticsClient%2BPaymentSheet.swift#L293) for safely logging errors but more lower level.